runelite-client/src/main/java/net/runelite/client/plugins/specialcounter/SpecialCounterPlugin.java

```diff
@@ -246,4 +268,19 @@ private int getHit(SpecialWeapon specialWeapon, int deltaExperience)
 			return (int) Math.round(damageOutput);
 		}
 	}
+
+	private void notifySpecialChange(int special)
+	{
+		// Not enabled, break
+		if (!config.enableSpecialAttackNotification())
+		{
+			return;
+		}
+		// Notify if we are now 100% and the last updated percentage was not default, 0%, or 100%
+		// Logging in with full special simultaneously triggers 0% and 100%
+		if (special == 1000 && this.specialPercentage > 0 && this.specialPercentage < 1000)
```

Comment:
As suggested in the issue of which this PR is based. This will also trigger when restoring from the house pool. Would suggest only notifying if the user has restored their special attack according to the natural restore rate. This would probably still cause some cases where the notification is sent when you are restoring at 90%-95% in your house pool but it would eliminate all the other instances.